src/lib/state.ts

```diff
@@ -50,6 +51,16 @@ export const dumpValue = (
   return valueState;
 };
 
+export const dumpValueNotification = (
```

Comment:
If this is only used in the forwarder and it depends on the `valueArgs`, there is no need to extract it in a function.

Reply:
I wanted the object to have the same signature as ValueState. I've now changed it so it still uses the same dumpValue function.